packages/core-data/README.md

```diff
@@ -90,58 +90,6 @@ _Returns_
 
 -   `Object`: Action object.
 
-### receiveAutosaves
```

Comment:
It looks like we print the same content in `@wordpress/core-data`'s README file. Should we link to the reference gude instead?